staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -693,6 +693,8 @@ type APIResource struct {
 	Verbs Verbs `json:"verbs" protobuf:"bytes,4,opt,name=verbs"`
 	// shortNames is a list of suggested short names of the resource.
 	ShortNames []string `json:"shortNames,omitempty" protobuf:"bytes,5,rep,name=shortNames"`
+	// groupNames is a list of group aliases this resource belongs to (e.g. 'all')
+	GroupNames []string `json:"groupNames,omitempty" protobuf:"bytes,6,rep,name=groupNames"`
```

Comment:
AliasMembership? Aliases? AliasInclusion? AliasesAssociated?

Definitely this can't be called anything with the word "group", too confusing with api groups.